docs/src/main/sphinx/connector/teradata.md

```diff
@@ -0,0 +1,334 @@
+
+
+### Teradata to Trino type mapping
+The connector maps Teradata types to the corresponding Trino types following this table:
+
+
+* - Teradata database type
+  - Trino type
+
+* - BYTEINT
+  - TINYINT
+
+* - SMALLINT
+  - SMALLINT
+
+* - INTEGER
+  - INTEGER
+
+* - BIGINT
+  - BIGINT
+
+* - DECIMAL(p,s)
+  - DECIMAL(p,s)
+
+* - NUMERIC(p,s)
+  - DECIMAL(p,s)
+
+* - FLOAT
+  - DOUBLE
+
+* - REAL
+  - REAL
+
+* - DOUBLE PRECISION
+  - DOUBLE
+
+* - CHAR(n)
+  - CHAR(n)
+
+* - VARCHAR(n)
+  - VARCHAR(n)
+
+* - LONG VARCHAR
+  - VARCHAR
+
+* - CLOB
+  - VARCHAR
+
+* - BYTE(n)
+  - VARBINARY
+
+* - VARBYTE(n)
+  - VARBINARY
+
+* - BLOB
+  - VARBINARY
+
+* - DATE
+  - DATE
+
+* - TIME
+  - TIME
+
+* - TIME WITH TIME ZONE
+  - TIME WITH TIME ZONE
+
+* - TIMESTAMP
+  - TIMESTAMP
+
+* - TIMESTAMP WITH TIME ZONE
+  - TIMESTAMP WITH TIME ZONE
+
+* - INTERVAL YEAR TO MONTH
+  - INTERVAL YEAR TO MONTH
+
+* - INTERVAL DAY TO SECOND
+  - INTERVAL DAY TO SECOND
+
+* - JSON
+  - JSON
+
+No other types are supported.
+
+
+### Trino to Teradata type mapping
+The connector maps Trino types to the corresponding Teradata types following this table:
+
+
+* - Trino type
+  - Teradata type
+
+* - TINYINT
+  - BYTEINT
+
+* - SMALLINT
+  - SMALLINT
+
+* - INTEGER
+  - INTEGER
+
+* - BIGINT
+  - BIGINT
+
+* - DECIMAL(p,s)
+  - DECIMAL(p,s)
+
+* - REAL
+  - REAL
+
+* - DOUBLE
+  - DOUBLE PRECISION
+
+* - CHAR(n)
+  - CHAR(n)
+
+* - VARCHAR(n)
+  - VARCHAR(n)
+
+* - VARBINARY
+  - VARBYTE
+
+* - DATE
+  - DATE
+
+* - TIME
+  - TIME
+
+* - TIME WITH TIME ZONE
+  - TIME WITH TIME ZONE
+
+* - TIMESTAMP
+  - TIMESTAMP
+
+* - TIMESTAMP WITH TIME ZONE
+  - TIMESTAMP WITH TIME ZONE
+
+* - JSON
+  - JSON
+
+No other types are supported.
+
+
+## Querying Teradata
+The Teradata connector provides a schema for every Teradata database. You can see the available Teradata databases by running SHOW SCHEMAS:
+```
+SHOW SCHEMAS FROM teradata;
+```
+If you have a Teradata database named sales, you can view the tables in this database by running SHOW TABLES:
+```
+SHOW TABLES FROM teradata.sales;
+```
+You can see a list of the columns in the orders table in the sales database using either of the following:
+```
+DESCRIBE teradata.sales.orders;
+SHOW COLUMNS FROM teradata.sales.orders;
+```
+Finally, you can access the orders table in the sales database:
+```
+SELECT * FROM teradata.sales.orders;
+```
+## SQL support
+The connector provides read access to data and metadata in the Teradata database. In addition to the [globally available](https://trino.io/docs/current/language/sql-support.html#globally-available-statements) and [read operation](https://trino.io/docs/current/language/sql-support.html#read-operations) statements, the connector supports the following features:
+
+## Procedures
+The connector supports the following procedures:
```

Comment:
Teradata connector doesn't support running procedures.  Please remove this section